PWGHF/D2H/Tasks/taskLb.cxx

```diff
@@ -34,33 +35,91 @@ using namespace o2::framework::expressions;
 
 void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
 {
-  ConfigParamSpec optionDoMC{"doMC", VariantType::Bool, true, {"Fill MC histograms."}};
+  ConfigParamSpec optionDoMC{"doMC", VariantType::Bool, false, {"Fill MC histograms."}};
```

Comment:
Hi Vit. This is from the previous version of the file. I tried to keep some of it for when I update the MC part. I see in the more modern files, that the MC switch is usually done with a PROCESS_SWITCH, which I am not familiar yet. Can I just make it a Configurable<bool> for now and keep the setup with switching in the workflow function?

Reply:
You need to replace the MC device with a process function `processMc`. Otherwise, you cannot run on MC on Hyperloop. You cannot switch a process function on/off with a `Configurable`. You need `PROCESS_SWITCH` for that.